src/ForecastDisplay.js

```diff
@@ -0,0 +1,38 @@
+import weatherDisplay from "./WeatherDisplay";
+import WeatherData from './WeatherData';
+
+//class used to display 4days forecast
+//to use it first declare API 
+//pass api reference and id of container in which you want to display forecast
+//to display forecast call function display(city) and pass name of the city as a string
+
+export default class ForecastDisplay{
+    constructor(api, containerId){
+        this.api=api;
+        this.containerId=containerId;
+        }
+    
+
+    mapData(data){
+        let fore = new Array();
+        fore = data.list.map(function(x) {
+            let res = new WeatherData(x); 
+            res.city = data.city.name;
+            res.sunrise = data.city.sunrise;
+            res.sunset = data.city.sunset; 
+            return res;});
+        return fore;
+   
+    }
```

Comment:
Metoda mapująca odpowiedź zewnętrznego API na klasę reprezentującą dane w apce bardziej pasowałaby w
 - klasie pobierającej dane z API
 - w mapperze stworzonym dokładnie po to
albo
 - w samej klasie reprezentującej te dane

W samej klasie wyświetlającej to nie całkiem pasuje (tutaj raczej jakieś formatowanie tych danych, przerabianie na stringi, które chce się konkretnie wyświetlić itp.)